packages/webrtc/src/api.ts

```diff
@@ -0,0 +1,179 @@
+//
```

Comment:
@sumboid Could you please check #55, should we have some unified json RPC stuff for both server APIs.

Reply:
Yeah I had a thought about that, let's discuss this on meeting: I think to use same client we need to rework `webrtc-service` a bit